src/go/accessd/server/server.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 The Magma Authors.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"context"
+	"net"
+	"time"
+
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+
+	"github.com/magma/magma/log"
+	"github.com/magma/magma/protos/magma/sctpd"
+	service_sctpd "github.com/magma/magma/service/sctpd"
+)
+
+const (
+	mmeUpstreamTarget  = "unix:///tmp/mme_sctpd_upstream.sock"
+	mmeGrpcDialTimeout = time.Second
+)
+
+func newMmeGrpcConn() (*grpc.ClientConn, error) {
+	ctx, _ := context.WithTimeout(context.Background(), mmeGrpcDialTimeout)
+	return grpc.DialContext(ctx, mmeUpstreamTarget, grpc.WithInsecure(), grpc.WithBlock())
+}
+
+func newServer(logger log.Logger) *grpc.Server {
+	grpcServer := grpc.NewServer()
+
+	mmeGrpcConn, err := newMmeGrpcConn()
+	if err != nil {
+		panic(err)
```

Comment:
I removed the log.fatal func because 1. it should be used rarely, 2. IMO explicit panic is more clear, 3. panic already dumps a stack and prints the values; usually no need for more machinery.

Once we have crash handling, sentry.io should grab these and send them up; I don't think adding a log package in the middle helps too much :)

Reply:
Will people notice easily in our log outputs - that a panic occurred? I worry they may need to synthesize multiple streams of data (e.g. sentry.io + logs) to know what happened when? Or no.

Reply:
@mstre123 discussion here related to your question on the lack of Fatal on logging interface below.